lms/djangoapps/course_blocks/management/commands/tests/test_generate_course_blocks.py

```diff
@@ -43,6 +45,20 @@ def _assert_courses_in_block_cache(self, *course_keys):
         for course_key in course_keys:
             self.assertTrue(is_course_in_block_structure_cache(course_key, self.store))
 
+    def _assert_courses_not_in_block_storage(self, *course_keys):
+        """
+        Assert courses don't exist in course block storage.
+        """
+        for course_key in course_keys:
+            self.assertFalse(is_course_in_block_structure_storage(course_key, self.store))
+
+    def _assert_courses_in_block_storage(self, *course_keys):
+        """
+        Assert courses exist in course block storage.
+        """
+        for course_key in course_keys:
+            self.assertTrue(is_course_in_block_structure_storage(course_key, self.store))
```

Comment:
As tried offline, the test code that used the consolidated/DRYed helper ended up being less self-explanatory (since the name of the test method became generic and booleans were passed around).  So we decided to leave this as is.